src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -267,6 +272,24 @@ public override BoundNode VisitCall(BoundCall node)
             return rewrittenBoundCall;
         }
 
+        private static MethodSymbol AdjustMethodForBaseInterfaceCall(BoundExpression receiverOpt, MethodSymbol method)
+        {
+            if (receiverOpt != null && receiverOpt.Kind == BoundKind.BaseReference)
```

Comment:
Consider using the `?.` operator here for consistency with other parts of this change.